README.md

```diff
@@ -42,8 +42,8 @@ Every post is structured with two main sections - the `meta` and `content` secti
 - `published` - Publishing date of the blog post. Used also for sorting posts by date.
 - `author` - Unique slug of the author (from Storyblok) usually looks like: `jon-doe`. Based on this property blog post page will display proper info about author (and image).
 - `category` - Array of categories for example `[Machine Learning]`. All categories should be the same as in the previously mentioned `categories.json` file. Important note: Categories are case sensitive.
-- `featuredImage` - Object with properties: `src` and `alt`. The `src` property is a path to featured image which is displayed on the posts list on the`/blog` page. The `alt` property is alternative text for the image. The image should be added to the `src/apps/labs/public/posts/<post-name>` directory, example: `src/apps/labs/public/posts/hello-world-post`. There is no need to provide full image path so the pathname should start with `/posts/`.
-- `hero` - Object with properties: `imageSrc` and `imageAlt`. The `imageSrc` property is a path to hero image which is displayed post page. The `imageAlt` property is alternative text for the image. The image should be added to the `src/apps/labs/public/posts/<post-name>` directory, example: `src/apps/labs/public/posts/hello-world-post`. There is no need to provide full image path so the pathname should start with `/posts/`.
+- `featuredImage` - Object with properties: `src` and `alt`. The `src` property is a path to featured image which is displayed on the posts list on the`/blog` page. The `alt` property is alternative text for the image. The image should be added to the `apps/labs/public/posts/<post-name>` directory, example: `apps/labs/public/posts/hello-world-post`. There is no need to provide full image path so the pathname should start with `/posts/`.
+- `hero` - Object with properties: `imageSrc` and `imageAlt`. The `imageSrc` property is a path to hero image which is displayed post page. The `imageAlt` property is alternative text for the image. The image should be added to the `apps/labs/public/posts/<post-name>` directory, example: `apps/labs/public/posts/hello-world-post`. There is no need to provide full image path so the pathname should start with `/posts/`.
```

Comment:
I am not sure about the date/folder structure question. 

Originally I wanted to migrate that, but then I saw the new structure here with the folder names for the images, and I thought that it might make things a bit weird to have a date nesting in the `apps/labs/post`s folder and a names nesting in the `apps/labs/public/posts`.

Alternatively, I can do the same dates nesting in both, and only in the end add the names nesting for the images? or even not have it and consider the month level sufficient?

What do you think?
@trallard maybe you want to chime in on this as well...

Reply:
@gabalafou is this a complex change? or can I just change the folder names and the label names and we're done?

Reply:
@trallard 
- do we in general want featured and hero to be different images? 
- for old posts should I use the existing illustration for both?
- for new posts will we have 2 illustration per post?
@gabalafou 
- do we need to size them differently if it's the same image, or can I point to the same image twice?

Reply:
I'm going to call it complex. 

In order to change the yaml data structure, we would have to update the source code. For example, we would have to update the [hero image type definition](https://github.com/Quansight/Quansight-website/blob/develop/libs/shared/ui-components/src/Hero/types.ts#L15), as well as [code that accesses properties of the hero image](https://github.com/Quansight/Quansight-website/blob/develop/libs/shared/ui-components/src/Hero/Hero.tsx#L12). We may in fact be limited by field names that Storyblok has already chosen for us. So if we want consistency, we may to change `featuredImage` and not `hero` in the yaml metadata.

I'm not sure what you're referring to when you say change the folder and label names.

As for the question of having two different images, I think this is something we want. The hero image is an image that precedes the blog post and is more of a widescreen landscape image, whereas the featured image appears in the list of blog posts and is more of a grid/square shape. The featured image is also what I imagine would appear on social media when the blog post is shared. I'm not a media specialist so I don't know the exact ratios here, but I think that for social you're looking at a ratio that's closer to 4:3, whereas our hero image can be more like 16:9. Correct me if I'm wrong, but that's how I see it.

Reply:
Thanks @gabalafou! 

- I will for now keep the labels as is. If we want to change the labels moving forward,
I would suggest to make that a post launch low-prio ticket. I think it's nice-to-have, 
and functionality is higher prio at the moment. **Agreed?**

- Since we don't have featured images for the old blog, I was thinking I would use the 
same images for both. 
**Would you suggest we crop/resize them for the feature image then?** 
    - If so, it would be useful to get a clear spec for the desired feature image, so that I can script 
a resize for a specific output instead of doing it manually. **Can you provide that, or should 
I ask Irina?**

Reply:
Totally. Agreed, the consistency of the yaml is not high priority.

I'm not sure about the images for the Labs blogs. I'm looking at the our website currently and a lot of blog posts have no images at all. I'm wondering if the blog index page is resilient to missing images—I mean, if it still looks good. 

One thing I would also do is see what happens when you try sharing a Labs blog article to a social media site like Facebook or Twitter. What image does their system pick? does it look good? I don't think we should have to go back and add images to blog posts that are missing them. Anyway this needs more investigation, but I'll have to get around to that later.

Reply:
Yes, Irina is aware that we need generic heros and features for our blogs without images and that I need to be able to have a way to change colors or otherwise introduce variation to the template design so I can create a large amount of images from her template.

I didn't think about the social media previews! good catch! I will have a sync with her and bring it up.